### PR TITLE
[qa] Upgraded to Django 2.0

### PR DESCRIPTION
**Pull request by phyxkal**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
-django>=1.11,<1.12
+django>=1.11,<2.1
 djangorestframework>=3.3,<3.8
 django-model-utils
 netdiff>=0.5.1,<=6.0
 openwisp-utils>=0.1.2,<0.2
 jsonfield
 six
+responses
```
Comment (rohithasrk) [inline requirements.txt]: @phyxkal This is already included in requirements-test.txt

Comment (rohithasrk) [reply requirements.txt]: You can remove the `responses` from here I suppose

Comment (phyxkal) [reply requirements.txt]: owh, okay

**Comment by nemesisdesign** [comment]
👍 !

**Pull request merged**